pegasus/test/test_section_api_helpers.rb

```diff
@@ -209,6 +209,22 @@ class SectionApiHelperTest < SequelTestCase
       end
     end
 
+    describe "fetch_user_sections" do
+      before do
+        FakeDashboard.use_fake_database
+      end
+
+      it 'returns returns all sections belonging to teacher' do
+        sections = DashboardSection.fetch_user_sections(FakeDashboard::TEACHER[:id])
+        assert_equal 3, sections.length
+      end
+
+      it 'specifies course for_id for sections that have one assigned' do
+        sections = DashboardSection.fetch_user_sections(FakeDashboard::TEACHER[:id])
+        assert_equal FakeDashboard::SECTION_COURSE[:course_id], sections[2][:course_id]
```

Comment:
Does `fetch_user_sections` have a contract on order (it appears not)? If not, ideally this would be robust to changes in the return order. Feel very welcome to punt, though.

Do you want to assert that the other sections have no `course_id`?